lib/ah/lograge/httparty/instrumentation.rb

```
@@ -0,0 +1,18 @@
+module Ah
+  module Lograge
+    module HTTParty
+      module Instrumentation
+        extend ActiveSupport::Concern
+
+        class_methods do
+          def perform_request(http_method, path, options, &block) #:nodoc:
+            uri = URI.parse(base_uri)
```

do we need to handle uri parse exceptions?

---

rather not, it won't fail silently anyway :-)